packages/renderer/src/lib/dashboard/ReleaseNotesBox.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+import { faCircleArrowUp } from '@fortawesome/free-solid-svg-icons';
+import { Button, CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+
+import Markdown from '../markdown/Markdown.svelte';
+
+let showBanner = false;
+let notesAvailable = false;
+let updateAvilable = false;
+let notesURL: string;
+let currentVersion: string;
+let title: string = '';
+let summary: string = '';
+let imageUrl: string = '';
+let imageAlt: string = '';
+
+window.events?.receive('show-release-notes', () => {
+  showBanner = true;
+});
+
+async function getInfoFromNotes() {
+  let curVersionSplit = currentVersion.split('.', 2);
+  const urlVersionFormat = curVersionSplit.join('.');
+  notesURL = `https://podman-desktop.io/release-notes/${urlVersionFormat}.json`;
+  const response = await fetch(notesURL);
+  if (!response.ok) {
+    notesURL = `https://github.com/containers/podman-desktop/releases/tag/v${currentVersion}`;
+  } else {
+    notesAvailable = true;
+    let notesInfo = await response.json();
+    imageUrl = notesInfo.image ? notesInfo.image : '';
+    imageAlt = imageUrl ? `Podman Desktop ${urlVersionFormat} release image` : '';
+    title = notesInfo.title ? notesInfo.title : '';
+    summary = notesInfo.summary ? notesInfo.summary : '';
+  }
+}
+
+function onclose() {
+  window.updateConfigurationValue(`releaseNotesBanner.show`, currentVersion);
+  showBanner = false;
+}
+
+onMount(async () => {
+  currentVersion = await window.getPodmanDesktopVersion();
+  showBanner = (await window.getConfigurationValue(`releaseNotesBanner.show`)) !== currentVersion ? true : false;
+  window
+    .podmanDesktopUpdateAvailable()
+    .then(available => (updateAvilable = available))
+    .catch(() => {
+      console.log('Cannot check for update');
+    });
+  await getInfoFromNotes();
+});
+</script>
+
+{#if showBanner}
+  {#if notesAvailable}
+    <div class="flex bg-[var(--pd-content-card-bg)] rounded-md p-5 gap-3 flex-row flex-nowrap h-[230px] items-center">
+      {#if imageUrl}
+        <img src={imageUrl} class="max-h-[190px] w-auto max-w-[20%] object-contain rounded-md" alt={imageAlt} />
+      {/if}
+      <div class="flex flex-col flex-1">
+        <div class="flex flex-row items-center justify-between">
+          <p class="text-[var(--pd-content-card-header-text)] font-bold text-xl ml-2">
+            {title}
+          </p>
+          <CloseButton on:click={() => onclose()} />
```

Comment:
```suggestion
          <CloseButton on:click={onclose} />
```

as we have/will have an eslint rule that enforce return type on functions/lamba declarations and Svelte does not support lamba with defined return type

Also `onclose` shoud be renamed to `onClose`